package.json

```diff
@@ -26,8 +26,14 @@
     "url": "https://github.com/blurymind/tilemap-editor/issues"
   },
   "homepage": "https://github.com/blurymind/tilemap-editor#readme",
+  "dependencies": {
+    "serve-handler": "^6.1.3"
```

Comment:
can we also make this a dev-dependency?

Reply:
it needs to be a dependency, for the CLI to work in npx.

Reply:
One terrible hack might be to combine npx with serve, in the script, instead:

```js
#!/usr/bin/env node

const { spawn } = require('child_process')
process.chdir(__dirname)
const proc = spawn('npx', ["serve"])
proc.stdout.pipe(process.stdout)
```

You could go further and remove `serve` from `devDependencies` and set `start` to `npx serve`.

This seems like a dark-pattern though, in order to not have a dependency listed (calling npx, inside a potential npx call) and since the only people who are running this command (vs using the deployed copy, at a URL) should probably install it's deps properly via `npm` (using the `dependencies` field) I think the other way is a bit better.

Reply:
can we not simply do this
http://www.sheshbabu.com/posts/publishing-npx-command-to-npm/
why even have a serve handler? :)

Reply:
> can we not simply do this

if you look at that link, you will see you need to make a javascript handler that does what you want (that you put in the `bin` field) which is what I did. That handler needs serve as a dependency.

> I am assuming npx tilemap-editor@latest does that.

No.

> seems easier to me to visit https://blurymind.github.io/tilemap-editor/

Yeh, agreed. That is another way to run the editor, if you don't want to run it locally.

> and even install it so it works offline, than to run npx tilemap-editor@latest

Not sure what you mean. Try that. Does it do what you expect? Without a handler, it doesn't run anything, so there is no local web-server.

> If that is the case, there is no justification to add a dependency to it imo

Well, that's up to you. Seems like a weird choice to me. Like the runtime code has no added dependency, it's just so the `npx` command works. If you don't want the `npx` command to work, then don't do it, but that was the point of this PR.

Reply:
If not having a dependency listed is so important, I offer a solution above. I don't think it's good (listing the dependency is better) but that would resolve the "I don't want a dependency listed"